### ISSUE TITLE
building a decompilated apk without any changes, Error on Manifest(not well-formed invalid token)

### ISSUE THREAD

**Issue by FelipeCardona95** [iBotPeaches/Apktool#2211]
### Information
1. **Apktool Version (`apktool -version`)** - 2.4.0
2. **Operating System (Mac, Linux, Windows)** -Windows 10
3. **APK From? (Playstore, ROM, Other)** - ApkPure

Include stacktrace here
```
`C:\Users\Felipe Cardona\Desktop\jadx\bin>apktool b supers2tar_test
I: Using Apktool 2.4.0
I: Checking whether sources has changed...
I: Smaling smali folder into classes.dex...
I: Checking whether resources has changed...
I: Building resources...
S: WARNING: Could not write to (C:\Users\Felipe Cardona\AppData\Local\apktool\framework), using C:\Users\FELIPE~1\AppData\Local\Temp\ instead...
S: Please be aware this is a volatile directory and frameworks could go missing, please utilize --frame-path if the default storage directory is unavailable
W: C:\Users\Felipe Cardona\Desktop\jadx\bin\supers2tar_test\AndroidManifest.xml:23: error: Error parsing XML: not well-formed (invalid token)
W:
brut.androlib.AndrolibException: brut.common.BrutException: could not exec (exit code = 1): [C:\Users\FELIPE~1\AppData\Local\Temp\brut_util_Jar_4533904425007830213.tmp, p, --forced-package-id, 127, --min-sdk-version, 16, --target-sdk-version, 27, --version-code, 99, --version-name, 2.9.3, --no-version-vectors, -F, C:\Users\FELIPE~1\AppData\Local\Temp\APKTOOL3583740018194891599.tmp, -0, arsc, -0, META-INF/android.arch.lifecycle_runtime.version, -0, META-INF/android.arch.lifecycle_reactivestreams.version, -0, META-INF/com.android.support_support-core-ui.version, -0, META-INF/com.android.support_support-v4.version, -0, META-INF/com.android.support_support-vector-drawable.version, -0, META-INF/com.android.support_support-core-utils.version, -0, META-INF/com.android.support_support-fragment.version, -0, META-INF/android.arch.core_runtime.version, -0, META-INF/android.arch.lifecycle_extensions.version, -0, META-INF/com.android.support_cardview-v7.version, -0, META-INF/com.android.support_appcompat-v7.version, -0, META-INF/com.android.support_support-compat.version, -0, META-INF/com.android.support_support-media-compat.version, -0, RST, -0, DCR, -0, assets/bgm/bgm_cardlist.a.bytes, -0, assets/bgm/bgm_main.a.bytes, -0, assets/bgm/bgm_result.a.bytes, -0, assets/bgm/bgm_shop.a.bytes, -0, png, -0, assets/soundfx/bonus.a.bytes, -0, assets/soundfx/change_theme.a.bytes, -0, assets/soundfx/excellent.a.bytes, -0, assets/soundfx/exp_loop.a.bytes, -0, assets/soundfx/fever.a.bytes, -0, assets/soundfx/gameover.a.bytes, -0, assets/soundfx/heart_on.a.bytes, -0, assets/soundfx/loadingcheer.a.bytes, -0, assets/soundfx/missionfailed.a.bytes, -0, assets/soundfx/normal_box.a.bytes, -0, assets/soundfx/perfectstage.a.bytes, -0, assets/soundfx/powerupsuccess.a.bytes, -0, assets/soundfx/premium_box.a.bytes, -0, assets/soundfx/rave1.a.bytes, -0, assets/soundfx/rave4.a.bytes, -0, assets/soundfx/resultcheer.a.bytes, -0, assets/soundfx/resultscore1s.a.bytes, -0, assets/soundfx/resultscore2s.a.bytes, -0, assets/soundfx/resumebeep.a.bytes, -0, assets/soundfx/star_t.a.bytes, -0, assets/soundfx/start.a.bytes, -0, assets/soundfx/weekly_league_top.a.bytes, -0, assets/ss/image/emblem/13.i.bytes, -0, assets/ss/image/emblem/14.i.bytes, -0, assets/ss/image/emblem/15.i.bytes, -0, assets/ss/image/emblem/20.i.bytes, -0, assets/ss/image/emblem/22.i.bytes, -0, assets/ss/image/emblem/26.i.bytes, -0, assets/ss/image/emblem/34.i.bytes, -0, assets/ss/image/emblem/38.i.bytes, -0, assets/ss/image/emblem/99.i.bytes, -0, assets/ss/image/emblem/9999.i.bytes, -0, assets/ss/image/icon/201.i.bytes, -0, assets/ss/image/icon/202.i.bytes, -0, assets/ss/image/icon/203.i.bytes, -0, assets/ss/image/icon/204.i.bytes, -0, assets/ss/image/icon/205.i.bytes, -0, assets/ss/image/icon/206.i.bytes, -0, assets/ss/image/icon/207.i.bytes, -0, assets/ss/image/icon/208.i.bytes, -0, assets/ss/image/icon/209.i.bytes, -0, assets/ss/image/icon/210.i.bytes, -0, assets/ss/image/icon/211.i.bytes, -0, assets/ss/image/icon/212.i.bytes, -0, assets/ss/image/icon/213.i.bytes, -0, assets/ss/image/icon/215.i.bytes, -0, assets/ss/image/icon/217.i.bytes, -0, webm, -0, assets/sync/sync.a.bytes, -0, res/drawable-hdpi-v4/abc_ab_share_pack_mtrl_alpha.9.png, -0, res/drawable-hdpi-v4/abc_btn_switch_to_on_mtrl_00001.9.png, -0, res/drawable-hdpi-v4/abc_btn_switch_to_on_mtrl_00012.9.png, -0, res/drawable-hdpi-v4/abc_cab_background_top_mtrl_alpha.9.png, -0, res/drawable-hdpi-v4/abc_list_divider_mtrl_alpha.9.png, -0, res/drawable-hdpi-v4/abc_list_focused_holo.9.png, -0, res/drawable-hdpi-v4/abc_list_longpressed_holo.9.png, -0, res/drawable-hdpi-v4/abc_list_pressed_holo_dark.9.png, -0, res/drawable-hdpi-v4/abc_list_pressed_holo_light.9.png, -0, res/drawable-hdpi-v4/abc_list_selector_disabled_holo_dark.9.png, -0, res/drawable-hdpi-v4/abc_list_selector_disabled_holo_light.9.png, -0, res/drawable-hdpi-v4/abc_menu_hardkey_panel_mtrl_mult.9.png, -0, res/drawable-hdpi-v4/abc_popup_background_mtrl_mult.9.png, -0, res/drawable-hdpi-v4/abc_scrubber_primary_mtrl_alpha.9.png, -0, res/drawable-hdpi-v4/abc_scrubber_track_mtrl_alpha.9.png, -0, res/drawable-hdpi-v4/abc_spinner_mtrl_am_alpha.9.png, -0, res/drawable-hdpi-v4/abc_switch_track_mtrl_alpha.9.png, -0, res/drawable-hdpi-v4/abc_tab_indicator_mtrl_alpha.9.png, -0, res/drawable-hdpi-v4/abc_textfield_activated_mtrl_alpha.9.png, -0, res/drawable-hdpi-v4/abc_textfield_default_mtrl_alpha.9.png, -0, res/drawable-hdpi-v4/abc_textfield_search_activated_mtrl_alpha.9.png, -0, res/drawable-hdpi-v4/abc_textfield_search_default_mtrl_alpha.9.png, -0, res/drawable-hdpi-v4/com_facebook_tooltip_black_background.9.png, -0, res/drawable-hdpi-v4/com_facebook_tooltip_blue_background.9.png, -0, res/drawable-hdpi-v4/common_google_signin_btn_icon_dark_normal_background.9.png, -0, res/drawable-hdpi-v4/common_google_signin_btn_icon_light_normal_background.9.png, -0, res/drawable-hdpi-v4/common_google_signin_btn_text_dark_normal_background.9.png, -0, res/drawable-hdpi-v4/common_google_signin_btn_text_light_normal_background.9.png, -0, res/drawable-hdpi-v4/notification_bg_low_normal.9.png, -0, res/drawable-hdpi-v4/notification_bg_low_pressed.9.png, -0, res/drawable-hdpi-v4/notification_bg_normal.9.png, -0, res/drawable-hdpi-v4/notification_bg_normal_pressed.9.png, -0, res/drawable-ldrtl-hdpi-v17/abc_spinner_mtrl_am_alpha.9.png, -0, res/drawable-ldrtl-mdpi-v17/abc_spinner_mtrl_am_alpha.9.png, -0, res/drawable-ldrtl-xhdpi-v17/abc_spinner_mtrl_am_alpha.9.png, -0, res/drawable-ldrtl-xxhdpi-v17/abc_spinner_mtrl_am_alpha.9.png, -0, res/drawable-ldrtl-xxxhdpi-v17/abc_spinner_mtrl_am_alpha.9.png, -0, res/drawable-mdpi-v4/abc_ab_share_pack_mtrl_alpha.9.png, -0, res/drawable-mdpi-v4/abc_btn_switch_to_on_mtrl_00001.9.png, -0, res/drawable-mdpi-v4/abc_btn_switch_to_on_mtrl_00012.9.png, -0, res/drawable-mdpi-v4/abc_cab_background_top_mtrl_alpha.9.png, -0, res/drawable-mdpi-v4/abc_list_divider_mtrl_alpha.9.png, -0, res/drawable-mdpi-v4/abc_list_focused_holo.9.png, -0, res/drawable-mdpi-v4/abc_list_longpressed_holo.9.png, -0, res/drawable-mdpi-v4/abc_list_pressed_holo_dark.9.png, -0, res/drawable-mdpi-v4/abc_list_pressed_holo_light.9.png, -0, res/drawable-mdpi-v4/abc_list_selector_disabled_holo_dark.9.png, -0, res/drawable-mdpi-v4/abc_list_selector_disabled_holo_light.9.png, -0, res/drawable-mdpi-v4/abc_menu_hardkey_panel_mtrl_mult.9.png, -0, res/drawable-mdpi-v4/abc_popup_background_mtrl_mult.9.png, -0, res/drawable-mdpi-v4/abc_scrubber_primary_mtrl_alpha.9.png, -0, res/drawable-mdpi-v4/abc_scrubber_track_mtrl_alpha.9.png, -0, res/drawable-mdpi-v4/abc_spinner_mtrl_am_alpha.9.png, -0, res/drawable-mdpi-v4/abc_switch_track_mtrl_alpha.9.png, -0, res/drawable-mdpi-v4/abc_tab_indicator_mtrl_alpha.9.png, -0, res/drawable-mdpi-v4/abc_textfield_activated_mtrl_alpha.9.png, -0, res/drawable-mdpi-v4/abc_textfield_default_mtrl_alpha.9.png, -0, res/drawable-mdpi-v4/abc_textfield_search_activated_mtrl_alpha.9.png, -0, res/drawable-mdpi-v4/abc_textfield_search_default_mtrl_alpha.9.png, -0, res/drawable-mdpi-v4/com_facebook_tooltip_black_background.9.png, -0, res/drawable-mdpi-v4/com_facebook_tooltip_blue_background.9.png, -0, res/drawable-mdpi-v4/common_google_signin_btn_icon_dark_normal_background.9.png, -0, res/drawable-mdpi-v4/common_google_signin_btn_icon_light_normal_background.9.png, -0, res/drawable-mdpi-v4/common_google_signin_btn_text_dark_normal_background.9.png, -0, res/drawable-mdpi-v4/common_google_signin_btn_text_light_normal_background.9.png, -0, res/drawable-mdpi-v4/notification_bg_low_normal.9.png, -0, res/drawable-mdpi-v4/notification_bg_low_pressed.9.png, -0, res/drawable-mdpi-v4/notification_bg_normal.9.png, -0, res/drawable-mdpi-v4/notification_bg_normal_pressed.9.png, -0, res/drawable-xhdpi-v4/abc_ab_share_pack_mtrl_alpha.9.png, -0, res/drawable-xhdpi-v4/abc_btn_switch_to_on_mtrl_00001.9.png, -0, res/drawable-xhdpi-v4/abc_btn_switch_to_on_mtrl_00012.9.png, -0, res/drawable-xhdpi-v4/abc_cab_background_top_mtrl_alpha.9.png, -0, res/drawable-xhdpi-v4/abc_list_divider_mtrl_alpha.9.png, -0, res/drawable-xhdpi-v4/abc_list_focused_holo.9.png, -0, res/drawable-xhdpi-v4/abc_list_longpressed_holo.9.png, -0, res/drawable-xhdpi-v4/abc_list_pressed_holo_dark.9.png, -0, res/drawable-xhdpi-v4/abc_list_pressed_holo_light.9.png, -0, res/drawable-xhdpi-v4/abc_list_selector_disabled_holo_dark.9.png, -0, res/drawable-xhdpi-v4/abc_list_selector_disabled_holo_light.9.png, -0, res/drawable-xhdpi-v4/abc_menu_hardkey_panel_mtrl_mult.9.png, -0, res/drawable-xhdpi-v4/abc_popup_background_mtrl_mult.9.png, -0, res/drawable-xhdpi-v4/abc_scrubber_primary_mtrl_alpha.9.png, -0, res/drawable-xhdpi-v4/abc_scrubber_track_mtrl_alpha.9.png, -0, res/drawable-xhdpi-v4/abc_spinner_mtrl_am_alpha.9.png, -0, res/drawable-xhdpi-v4/abc_switch_track_mtrl_alpha.9.png, -0, res/drawable-xhdpi-v4/abc_tab_indicator_mtrl_alpha.9.png, -0, res/drawable-xhdpi-v4/abc_textfield_activated_mtrl_alpha.9.png, -0, res/drawable-xhdpi-v4/abc_textfield_default_mtrl_alpha.9.png, -0, res/drawable-xhdpi-v4/abc_textfield_search_activated_mtrl_alpha.9.png, -0, res/drawable-xhdpi-v4/abc_textfield_search_default_mtrl_alpha.9.png, -0, res/drawable-xhdpi-v4/com_facebook_tooltip_black_background.9.png, -0, res/drawable-xhdpi-v4/com_facebook_tooltip_blue_background.9.png, -0, res/drawable-xhdpi-v4/common_google_signin_btn_icon_dark_normal_background.9.png, -0, res/drawable-xhdpi-v4/common_google_signin_btn_icon_light_normal_background.9.png, -0, res/drawable-xhdpi-v4/common_google_signin_btn_text_dark_normal_background.9.png, -0, res/drawable-xhdpi-v4/common_google_signin_btn_text_light_normal_background.9.png, -0, res/drawable-xhdpi-v4/notification_bg_low_normal.9.png, -0, res/drawable-xhdpi-v4/notification_bg_low_pressed.9.png, -0, res/drawable-xhdpi-v4/notification_bg_normal.9.png, -0, res/drawable-xhdpi-v4/notification_bg_normal_pressed.9.png, -0, res/drawable-xxhdpi-v4/abc_ab_share_pack_mtrl_alpha.9.png, -0, res/drawable-xxhdpi-v4/abc_btn_switch_to_on_mtrl_00001.9.png, -0, res/drawable-xxhdpi-v4/abc_btn_switch_to_on_mtrl_00012.9.png, -0, res/drawable-xxhdpi-v4/abc_cab_background_top_mtrl_alpha.9.png, -0, res/drawable-xxhdpi-v4/abc_list_divider_mtrl_alpha.9.png, -0, res/drawable-xxhdpi-v4/abc_list_focused_holo.9.png, -0, res/drawable-xxhdpi-v4/abc_list_longpressed_holo.9.png, -0, res/drawable-xxhdpi-v4/abc_list_pressed_holo_dark.9.png, -0, res/drawable-xxhdpi-v4/abc_list_pressed_holo_light.9.png, -0, res/drawable-xxhdpi-v4/abc_list_selector_disabled_holo_dark.9.png, -0, res/drawable-xxhdpi-v4/abc_list_selector_disabled_holo_light.9.png, -0, res/drawable-xxhdpi-v4/abc_menu_hardkey_panel_mtrl_mult.9.png, -0, res/drawable-xxhdpi-v4/abc_popup_background_mtrl_mult.9.png, -0, res/drawable-xxhdpi-v4/abc_scrubber_primary_mtrl_alpha.9.png, -0, res/drawable-xxhdpi-v4/abc_scrubber_track_mtrl_alpha.9.png, -0, res/drawable-xxhdpi-v4/abc_spinner_mtrl_am_alpha.9.png, -0, res/drawable-xxhdpi-v4/abc_switch_track_mtrl_alpha.9.png, -0, res/drawable-xxhdpi-v4/abc_tab_indicator_mtrl_alpha.9.png, -0, res/drawable-xxhdpi-v4/abc_textfield_activated_mtrl_alpha.9.png, -0, res/drawable-xxhdpi-v4/abc_textfield_default_mtrl_alpha.9.png, -0, res/drawable-xxhdpi-v4/abc_textfield_search_activated_mtrl_alpha.9.png, -0, res/drawable-xxhdpi-v4/abc_textfield_search_default_mtrl_alpha.9.png, -0, res/drawable-xxhdpi-v4/common_google_signin_btn_icon_dark_normal_background.9.png, -0, res/drawable-xxhdpi-v4/common_google_signin_btn_icon_light_normal_background.9.png, -0, res/drawable-xxhdpi-v4/common_google_signin_btn_text_dark_normal_background.9.png, -0, res/drawable-xxhdpi-v4/common_google_signin_btn_text_light_normal_background.9.png, -0, res/drawable-xxxhdpi-v4/abc_btn_switch_to_on_mtrl_00001.9.png, -0, res/drawable-xxxhdpi-v4/abc_btn_switch_to_on_mtrl_00012.9.png, -0, res/drawable-xxxhdpi-v4/abc_spinner_mtrl_am_alpha.9.png, -0, res/drawable-xxxhdpi-v4/abc_switch_track_mtrl_alpha.9.png, -0, res/drawable-xxxhdpi-v4/abc_tab_indicator_mtrl_alpha.9.png, -0, arsc, -I, C:\Users\FELIPE~1\AppData\Local\Temp\1.apk, -S, C:\Users\Felipe Cardona\Desktop\jadx\bin\supers2tar_test\res, -M, C:\Users\Felipe Cardona\Desktop\jadx\bin\supers2tar_test\AndroidManifest.xml]`

pastebin: https://pastebin.com/DMaACNFS

### Steps to Reproduce
1. apktool d "SuperStar SMTOWN_v2.9.3_apkpure.com.apk" -o supers2tar_test
2. apktool b supers2tar_test

### APK
https://apkpure.com/es/superstar-smtown/kr.co.dalcomsoft.superstar.a

### Questions to ask before submission
1. Have you tried `apktool d`, `apktool b` without changing anything? Yes
2. If you are trying to install a modified apk, did you resign it? No
3. Are you using the latest apktool version? Yes

**Comment by BullerEX** [comment]
Its related to the resources, if I do the d -r and then b it creates the app without problem, but not deconding the res wont make the apk break afterwards?

**Comment by iBotPeaches** [comment]
```
W: C:\Users\Felipe Cardona\Desktop\jadx\bin\supers2tar_test\AndroidManifest.xml:23: error: Error parsing XML: not well-formed (invalid token)
```

This was probably fixed with our namespace changes in v2.5.0 to prevent garbled attributes. I don't have an apk to confirm exactly, so going without enough details to triage.

Sorry for delay.

**Comment by aravi365** [comment]
Not yet fixed. getting the same error on 2.60 as wekk
